### PR TITLE
fix(iOS): crash on reload

### PR DESCRIPTION
**Pull request by hirbod**
### Summary

This PR fixes a crash in `viewControllerDidChangeWidth` caused by `nil` values during React Native reloads.

### Changes

Applied optional chaining to handle `nil` values for `bridge` and `bridge.uiManager`.

### CODE REVIEW

File: ios/TrueSheetView.swift [hunk 1]
```diff
@@ -19,7 +19,7 @@ class TrueSheetView: UIView, RCTInvalidating, TrueSheetViewControllerDelegate {
 
   private var isPresented = false
   private var activeIndex: Int?
-  private var bridge: RCTBridge
+  private var bridge?: RCTBridge
```
Comment (lodev09) [inline ios/TrueSheetView.swift:22]: I'm getting error in xCode on this line.

This works
```
private var bridge: RCTBridge?
```

File: ios/TrueSheetView.swift [hunk 2]
```diff
@@ -137,7 +137,7 @@ class TrueSheetView: UIView, RCTInvalidating, TrueSheetViewControllerDelegate {
     guard let containerView else { return }
 
     let size = CGSize(width: width, height: containerView.bounds.height)
-    bridge.uiManager.setSize(size, for: containerView)
+    bridge?.uiManager?.setSize(size, for: containerView)
```
Comment (lodev09) [inline ios/TrueSheetView.swift:140]: You can omit `.uiManager?` optional here

**Review by lodev09 — COMMENTED** [review]
Minor comments

File: ios/TrueSheetView.swift (hunk 2 above)
Comment (hirbod) [reply ios/TrueSheetView.swift]: Done

File: ios/TrueSheetView.swift (hunk 1 above)
Comment (hirbod) [reply ios/TrueSheetView.swift]: Weird, not sure how this happened. Fixed

**Review by lodev09 — APPROVED** [review]
Goods 👍

**Comment by lodev09** [comment]
:rocket: _This pull request is included in v0.12.0. See [Release 0.12.0](https://github.com/lodev09/react-native-true-sheet/releases/tag/v0.12.0) for release notes._

**Comment by rranjan14** [comment]
I am using version `2.0.6` with expo@53. Still crashing on ios. I have created a repo where I am able to repro it. https://github.com/rranjan14/bottom-sheet-crash-repro

**Pull request merged**